atom/browser/browser.h

```diff
@@ -167,6 +167,13 @@ class Browser : public WindowListObserver {
   // one from app's name.
   // The returned string managed by Browser, and should not be modified.
   PCWSTR GetAppUserModelID();
+
+  typedef HRESULT (STDAPICALLTYPE *DwmGetColorizationColor)(DWORD *, BOOL *);
```

Comment:
What about fixing the build instead of adding complexity to the code? :) what was it complaining about?